src/js/themes/hpe.js

```diff
@@ -2881,6 +2871,14 @@ const buildTheme = (tokens, flags) => {
     },
     starRating: {
       color: 'background-selected-primary-strong',
```

Comment:
@Sulaymon333 you can test but I dont think you need this color in both here as well as defined in the icons